src/components/content.js

```diff
@@ -144,14 +144,35 @@ class Content extends React.Component {
   }
 
   /**
+   * The React ref method to set the root content element locally.
+   *
+   * @param {Element} n
+   */
+
+  ref = (n) => this.element = n
+
+  /**
+   * Check if an `event` is being fired from inside a non-contentediable child
+   * element, in which case we'll want to ignore it.
+   *
+   * @param {Event} event
+   * @return {Boolean}
+   */
+
+  isNonEditable = (event) => {
+    const { target } = event
+    return !target.isContentEditable || target !== this.element
```

Comment:
Yes, I thought about this too. My understanding is that this should only be used on events which are only triggered on input elements, including content-editables. The events which use `isInContentEditable()` are: `onBeforeInput`, `onBlur`, `onCompositionEnd`, `onCompositionStart`, `onCopy`, `onCut`, `onDragEnd`, `onDragOver`, `onDragStart`, `onDrop`, `onInput`, `onKeyDown`, `onKeyUp`, `onPaste`, `onSelect`.

Most of those will only target the input element, the only ones that may not be the case is the onDrag/Drop events? So perhaps we should be traversing the target's parent instead of only checking the target. It will definitely be safer.